aip/general/0149.md

```diff
@@ -38,3 +38,29 @@ message Book {
 value and unset most of the time; if an alternative design does not require
 such a distinction, it is usually preferred. In practice, this means `optional`
 **should** only ever be used for integers and floats.
+
+**Important:** Tracking field presence is *not* the same as documenting API
+field behavior as defined in [AIP-203][]. For example, a field labeled with
+`optional` for presence tracking **may** also be annotated as
+`google.api.field_behavior = REQUIRED` if the field must be set. If you only
+want to document the server perceived behavior of a field, read [AIP-203][].
+
+## Rationale
+
+### `optional` and field behavior
```

Comment:
Flip these. Or flip their references in the paragraph below.

IOW, this heading talks about `optional` first, but in the paragraph below, it refers optional as the latter.

Reply:
Good call. Done.